composer.json

```diff
@@ -100,5 +100,10 @@
         "psr-4": {
             "OpenEMR\\": "src"
         }
+    },
+    "autoload-dev": {
```

Comment:
I'm assuming we would want to provide a separate means of autoloading classes for test cases. We'll see how things go with the Travis CI :smile:

Basically I'm including the test root here so that I can utilize PHP's namespaces and "use" statements for importing classes rather than `require_once`.